code/API_definitions/location.yaml

```diff
@@ -97,11 +97,14 @@ components:
           $ref: '#/components/schemas/Longitude'
         accuracy:
           $ref: '#/components/schemas/Accuracy'
+        maxLocationAge:
+          $ref: '#/components/schemas/MaxLocationAge'
       required:
         - ueId
         - latitude
         - longitude
         - accuracy
+        - maxLocationAge 
```

Comment:
Maybe it should be optional, with a default value. It depends on how often it is expected to be used. If the common case is to request "real-time" info (less than a minute or few minutes), it is not convenient to ask for it in every request.

Reply:
A default value for maxLocationAge is a good idea. I vote for.